Documentation/disaster-recovery.md

```diff
@@ -106,3 +106,102 @@ kubectl create -f operator.yaml
 ```
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `monCount`.
+
+# Bring orphaned rook/ceph cluster back to Kubernetes
+
+## What this section trying to resolve
+
+1. You had a running rook/ceph cluster, suddenly your Kubernetes environment exploded, you have to start a new Kubernetes environment and put your existing rook/ceph cluster back.
+2. You are migrating your existing rook/ceph cluster to a new Kubernetes environment, downtime can be tolerated.
+
+In author's situation,  `etcd` data of running Kubernetes cluster is nuked and have no backup, all OSDs are using bluestore backend.
+
+### Prerequisites
+
+1. A working Kubernetes cluster without rook
```

Comment:
I was a bit confused at first here. I think additional context would be useful: "A working Kubernetes cluster to which we will migrate the previous Rook Ceph cluster."

It can still have Rook as long as it is a different Rook cluster (differently named and namespaced) than the one we are migrating.

Reply:
Replaced as suggestion